test/integration/lib/philipsHue_server.js

```diff
@@ -160,9 +102,112 @@ var philipsHue_server = (function() {
      });
    });
   }
+   
+  // Callback methods
+
+  _server.use(function (req, res, next) { 
+    // foxbox does not have the json header, so need to 
+    // inject it here so the message body can be properly captured
+    req.headers['content-type'] = 'application/json';
+    last_received_cmd = req.method + ':' + req.originalUrl;
+    next();
+  });
+
+  _server.get('/', function (req, res) {
+    res.status(200).sendFile(
+      path.join(__dirname,'/html/philips_initial.html'));  
+  });
+
+  _server.get('/api/:foxbox/', function (req, res) {   
```

Comment:
oh no actually, the foxbox is what the Foxbox sends as its username, which is always 'unauthorized'.  this gets checks in req.params.foxbox.